lib/finicity/v1/request/generate_finicity_connect_link.rb

```diff
@@ -27,10 +28,19 @@ def body
         data['customerId'] = "#{customer_id}"
         data['redirectUri'] = redirect_uri
         data['type'] = 'aggregation'
-        data['oauthOptions'] = {
-          'enabled' => oauth_support,
-          'autoReplace' => autoreplace
-        }
+        if oauth_support
+          if autoreplace
+            data['oauthOptions'] = {
+              'enabled' => oauth_support,
+              'autoReplace' => autoreplace
+            }
```

Comment:
You can actually use symbols here and they will still be converted to json string correctly:
```
data['oauthOptions'] = {
  enabled: oauth_support,
  autoReplace: autoreplace
}
```
This also applies to the code below.